okapi-core/src/main/java/org/folio/okapi/managers/TenantManager.java

```diff
@@ -1035,4 +1036,8 @@ private ModuleInstance getTenantInstanceForInterface(
     });
   }
 
+  @Override
+  public Future<Void> isAlive() {
+    return tenantStore.listTenants().mapEmpty();
```

Comment:
this includes supertenant, right?

Reply:
yes.

Reply:
isAlive() succeeds if the list is empty, contains the supertenant only, or contains any number of tenants.
If only fails if the database query fails.